lib/elixir/lib/access.ex

```diff
@@ -305,6 +305,32 @@ defmodule Access do
       "could not put/update key #{inspect key} on a nil value"
   end
 
+  @doc """
+  Removes the entry with a given key from a struct, map, or keyword list.
```

Comment:
The reason I made it explicit is that I felt container was too broad. We
really mean "thing that implements the Access behavior". So structs in
general are not usable here, but a struct that implements pop() is.



On Tue, Jul 5, 2016 at 2:00 PM, José Valim <notifications@github.com> wrote:

> In lib/elixir/lib/access.ex
> <https://github.com/elixir-lang/elixir/pull/4948#discussion_r69618885>:
>
> > @@ -305,6 +305,32 @@ defmodule Access do
> >        "could not put/update key #{inspect key} on a nil value"
> >    end
> >
> > +  @doc """
> > +  Removes the entry with a given key from a struct, map, or keyword list.
>
> Yes, we call container in the other functions, so 👍 for container here.
> :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/elixir-lang/elixir/pull/4948/files/f728a6c3425ce7d0a8549120f7f71d664e2696a1#r69618885>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAApmGX0W-GY4JLbXqUOfD0D3m0WFZ7hks5qSqm4gaJpZM4JFZ-i>
> .
>